db/migrate/20181111003546_remove_binary_builder.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```

Comment:
Should this be in the plugin's `db/migrate`?

Reply:
can't really be there since I'm deleting that plugin :D 
... but I guess it needs to not blow up when column is not there

Reply:
Ah 🤦‍♂️ makes sense